server/statistics/hot_peer_cache.go

```diff
@@ -54,24 +55,30 @@ var (
 
 // hotPeerCache saves the hot peer's statistics.
 type hotPeerCache struct {
-	kind           FlowKind
-	peersOfStore   map[uint64]*TopN               // storeID -> hot peers
-	storesOfRegion map[uint64]map[uint64]struct{} // regionID -> storeIDs
+	mu struct {
+		sync.RWMutex
```

Comment:
Due to the low probability of conflict,`Mutex` should have better performance than `RWMutex`.